pyproject.toml

```diff
@@ -15,7 +15,7 @@ authors = [
 ]
 description = "Filetracker caching file storage"
 readme = "README.md"
-requires-python = ">=3.7"
+requires-python = ">=3.9"
```

Comment:
why not 3.13 here?
And if we want to leave it `>=3.9` shouldn't we run test with 3.9 in addition to 3.13?